apps/api/src/apps/users/schema/users.gql

```diff
@@ -1,15 +1,23 @@
 type User {
-  # Example field (placeholder)
-  exampleField: Int
+  id: String!
+  username: String!
+  email: String!
+  dateTimeCreated: Date!
+  dateTimeUpdated: Date!
 }
 
 input CreateUserInput {
-  # Example field (placeholder)
-  exampleField: Int
+  username: String!
+  password: String!
+  name: String!
+  email: String!
 }
 
 input UpdateUserInput {
   id: Int!
+  name: String
+  username: String
+  password: String
 }
```

Comment:
Use this structure for update:

`input UpdateUserDataInput {
  name: String
  username: String
  password: String
  emailAddress: String
}

input UpdateUserInput {
  id: String!
  data: UpdateUserDataInput!
}`

Reply:
we can also use structure like this

input UpdateUserDataInput {
name: String
username: String
password: String
emailAddress: String
}


type Mutation {
  ***
  updateUser(id: String, updateUserInput: UpdateUserInput!): User!
  ***
}

Reply:
OK lets do that